python-lib/odata_plugin_common.py

```diff
@@ -0,0 +1,11 @@
+try:
+    from urllib.parse import urlparse, parse_qs, urlunsplit
+except:
+    from urlparse import urlparse, parse_qs, urlunsplit
```

Comment:
Maybe put a comment to precise which one is for which python version for maintainability

Reply:
👍 Done in [19efbaf](https://github.com/dataiku/dss-plugin-odata/pull/5/commits/19efbaf6995ffc773bf4f47bb6cb81679205ee1e)